velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp

```diff
@@ -236,4 +236,36 @@ TEST_F(MinMaxTest, initialValue) {
   testAggregations({row}, {}, {"max(c1)"}, "SELECT -1");
 }
 
+/// TODO: Add vectorfuzzer support for decimals to use existing
+/// MinMaxTest::doTest() call.
+/// https://github.com/facebookincubator/velox/issues/3756
+TEST_F(MinMaxTest, decimalMinMax) {
+  auto longType = DECIMAL(23, 4);
+  auto longDecimal = makeNullableLongDecimalFlatVector(
+      {buildInt128(10, 300),
+       buildInt128(-10, 300),
```

Comment:
Please add some values that are very close (as much close as can be represented) to the min and max values in these arrays to give an idea of how small apart a difference this can pick up.
e.g. values 9.99999 and 9.99998 are as close as it gets for decimal(6,5)